data/domain/alerts.tsx

```diff
@@ -275,17 +307,47 @@ const getPlayerAlerts = (player: Player, anvil: AnvilWrapper, playerObols: Obol[
             if (![2, 3, 7].some(id => (player.cardInfo?.getBonusForId(id) ?? 0) > 0)) {
                 alerts.push(new CardSetAlert(player, `${player.cardInfo?.getCardSetText()} isn't optimal lab`, player.cardInfo?.getCardSetIcon() ?? 'CardSet26'));
             }
+            if ((playerDivinityData.gods.length ?? 0) == 1 && playerDivinityData.gods[1].data.name == 'Arctis') {
+                alerts.push(new DivinityLinkedAlert(player, "Arctis god is useless while you're in lab"));
+            }
             break;
         case Activity.Skilling:
             if (![1, 2, 3, 7].some(id => (player.cardInfo?.getBonusForId(id) ?? 0) > 0)) {
                 alerts.push(new CardSetAlert(player, `${player.cardInfo?.getCardSetText()} isn't optimal skilling`, player.cardInfo?.getCardSetIcon() ?? 'CardSet26'));
             }
             break;
+        case Activity.Divinity:
+            if ((player.skills.get(SkillsIndex.Divinity)?.level ?? 0) >= 80 && (playerDivinityData.style.name ?? '') != 'Mindful') {
+                alerts.push(new DivinityStyleAlert(player, "You should use Mindful style"));
+            } else if ((player.skills.get(SkillsIndex.Divinity)?.level ?? 0) >= 60 && (playerDivinityData.style.name ?? '') != 'Zen') {
+                // If using Mantra on 8 or more players it'll be better than using Zen for EXP
+                if (!(divinityData.playerInfo.filter(info => (info.style.name ?? '') == 'Mantra').length >= 8 && (playerDivinityData.style.name ?? '') == 'Mantra')) {
+                    alerts.push(new DivinityStyleAlert(player, "You should use Zen style"));
+                }
+            } else if ((player.skills.get(SkillsIndex.Divinity)?.level ?? 0) >= 25) {
+                // If using Mantra on 7 or more players it'll be better than using Vitalic for EXP
+                if (!(divinityData.playerInfo.filter(info => (info.style.name ?? '') == 'Mantra').length >= 7 && (playerDivinityData.style.name ?? '') == 'Mantra')) {
+                    alerts.push(new DivinityStyleAlert(player, "You should use Vitalic style"));
+                }
+            } else if ((player.skills.get(SkillsIndex.Divinity)?.level ?? 0) >= 10 && (playerDivinityData.style.name ?? '') == 'Kinesis') {
+                alerts.push(new DivinityStyleAlert(player, "You should use Focus style"));
+            } else if ((player.skills.get(SkillsIndex.Divinity)?.level ?? 0) >= 5 && (playerDivinityData.style.name ?? '') == 'Kinesis') {
+                alerts.push(new DivinityStyleAlert(player, "You should use Chakra style"));
+            }            
+            break;
         case Activity.Unknown:
             alerts.push(new DoingNothingAlert(player));
             break;
     }
 
+    // Separate those two from above to avoid having it in multiple case
+    if (player.getActivityType() != Activity.Lab && (playerDivinityData.gods.length ?? 0) == 1 && playerDivinityData.gods[1].data.name == 'Goharut') {
```

Comment:
This code actually blows up for me on `playerDivinityData.gods[1].data.name` with 
```
Cannot read properties of undefined (reading 'data')
```

Reply:
I'm dumb, I used index 1 instead of 0 to get the first element of the array due to habit with the language I use at work x) (yeah it's a strange one)